src/arviz_plots/plots/ridgeplot.py

```diff
@@ -0,0 +1,505 @@
+                kwarg=["y_bottom", "y_top"]
+            )
+            shade_start = -0.1 if x == 0 else x - 0.6
+            xlim_labels = [-0.1, len(labels) - 0.9]
+            plot_collection.map(
+                fill_between_y,
+                "shade",
+                data=shade_data,
+                x=[shade_start, xlim_labels[1]],
+                coords={"column": "labels"},
+                ignore_aes=shade_ignore,
+                **shade_kwargs,
+            )
+            ci_global_min = x_range.min().to_array().min().item()
+            ci_global_max = x_range.max().to_array().max().item()
+            ci_range_extend = 0.1 * (ci_global_max - ci_global_min)
+            xlim_ridge = [ci_global_min - ci_range_extend, ci_global_max + ci_range_extend]
+            plot_collection.map(
+                fill_between_y,
+                "shade",
+                data=shade_data,
+                x=xlim_ridge,
+                coords={"column": "ridge"},
+                ignore_aes=shade_ignore,
+                **shade_kwargs,
+            )
+        _, lab_aes, lab_ignore = filter_aes(plot_collection, aes_map, "labels", sample_dims)
+        extra_ignore_aes = []
+        for aes_key in lab_aes:
+            if aes_key == "overlay":
+                continue
+            aes_ds = plot_collection.get_aes_as_dataset(aes_key)
+            if set(aes_ds.dims).difference(cumulative_label):
+                extra_ignore_aes.append(aes_key)
+        lab_aes = set(lab_aes).difference(extra_ignore_aes)
+        lab_ignore = set(lab_ignore).union(extra_ignore_aes)
+        lab_kwargs = labels_kwargs.copy()
+        if "color" not in lab_aes:
+            lab_kwargs.setdefault("color", "black")
+        if x == 0:
+            lab_kwargs.setdefault("horizontal_align", "left")
+        if x == len(labels) - 1:
+            lab_kwargs.setdefault("horizontal_align", "right")
+        plot_collection.map(
+            annotate_label,
+            f"{label.strip('_')}_label",
+            data=y,
+            x=x,
+            dim=None if label == "__variable__" else label,
+            subset_info=True,
+            coords={"column": "labels"},
+            ignore_aes=lab_ignore,
+            **lab_kwargs,
+        )
+        x += 1
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+    ticklabel_kwargs = copy(plot_kwargs.get("ticklabels", {}))
+    if ticklabel_kwargs is not False:
+        plot_bknd.xticks(
+            np.arange(len(labels)),
+            [label.strip("_") for label in labellable_dims if label in labels],
+            plot_collection.get_target(None, {"column": "labels"}),
+            **ticklabel_kwargs,
+        )
+
+    # plot kde
+    default_color = plot_bknd.get_default_aes("color", 1, {})[0]
+    # rescaling kde
+    print(f"\n kde density pre rescaled = {density!r}")
+    for var in density.data_vars:
+        density[var].loc[{"plot_axis": "y"}] = density[var].sel(plot_axis="y") ** rescale_kde
```

Comment:
This type of transformation should not be an option. kdes are already hard enough to interpret (even if they look simple and we believe we can read them with ease that is not the case). I would not add this as this makes it basically impossible to actually read anything from the kde other than the range and the position of the peak.

By rescaling I meant multiplying all kdes by a common scale factor in order to preserve all proportions in the curve but having the tallest one peak at height 1 (or something else, this could be user modified, similarly to the overlap control argument there is in current `plot_forest`).

```suggestion
    for var in density.data_vars:
        density.loc[{"plot_axis": "y"}] = density.sel(plot_axis="y") / density.sel(plot_axis="y").max().to_array().max() * ridge_height
```

where `ridge_height` regulates the height of the ridge (that is, the height of the tallest peak in the ridge). If `ridge_height=1` (or lower) there should be no overlap at all (when combined=True and there aren't multiple models) with higher values there will be some area overlap (>1 but smallish should mean area overlap but no edge overlap generally, >>1 will generally mean both are and edge overlap). Once both area and edge plotting are working, we should try several values like 1, 1.5, 2 and see with the example arviz datasets how each looks.